Blog/migrations/0003_auto_20180409_0857.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.0.3 on 2018-04-09 08:57
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('Blog', '0002_post_title'),
```

Comment:
How is this change related to the Google authentication feature? If it is not related, it should be put into a separate comment where the commit message must contain the reason for this change.